lib/happo/views/index.erb

```diff
@@ -15,7 +15,9 @@
       body {
         background-color: #fff;
         margin: 0;
-        pointer-events: none;
+        <% unless params[:debug] %>
+          pointer-events: none;
```

Comment:
It might be worth adding a comment here explaining why we do this in the first place.